src/kernel/cpu/binary_bcast_reduce_sum.cc

```diff
@@ -12,9 +12,9 @@ namespace kernel {
 #define REDUCER ReduceSum
 #define XPU kDLCPU
 
-EVAL(GEN_NDIM, GEN_DTYPE, GEN_TARGET, GEN_BINARY_OP, GEN_BCAST_DEFINE);
+EVAL(GEN_NDIM, GEN_DTYPE, GEN_TARGET, GEN_BINARY_OP, GEN_BCAST_DEFINE)
 EVAL(GEN_BACKWARD_MODE, GEN_NDIM, GEN_DTYPE, GEN_TARGET, GEN_BINARY_OP,
-     GEN_BACKWARD_BCAST_DEFINE);
+     GEN_BACKWARD_BCAST_DEFINE)
```

Comment:
Could you revert these kinds of changes? I'm afraid there will be too many conflicts when merging.

Reply:
Sure. I will checkout the old version of these files.

Reply:
Done